qiskit/tools/visualization/__init__.py

```diff
@@ -11,6 +11,7 @@
 
 from ._circuit_visualization import circuit_drawer, plot_circuit, generate_latex_source,\
     latex_circuit_drawer, matplotlib_circuit_drawer, qx_color_scheme
+from ._state_visualization import plot_state, plot_bloch_vector
```

Comment:
I think we should probably remove `plot_state` from here because it's going to be imported again by the block just added in #862